src/views/templates/details/hooks/useIsTemplateEditable.ts

```diff
@@ -0,0 +1,32 @@
+import { TemplateModel, V1Template } from '@kubevirt-ui/kubevirt-api/console';
+import { useAccessReview } from '@openshift-console/dynamic-plugin-sdk';
+
+import { isCommonVMTemplate } from '../../utils/utils';
+
+/**
+ * This hook checks the user capabilities of editing templates in a namespace.
+ * Checks user permissions and template characteristics.
+ * The hook does not check if the user has permissions for that particular template
+ * as this behavior would compromise performarce in pages were a lot of templates are involved like template list.
+ * @param template template to check
+ * @returns boolean value
+ */
+const useIsTemplateEditable = (template: V1Template) => {
+  const [canUpdateTemplate, canUpdateLoaded] = useAccessReview({
+    verb: 'update',
+    resource: TemplateModel.plural,
+    namespace: template?.metadata?.namespace,
+  });
+
+  const [canPatchTemplate, canPatchLoaded] = useAccessReview({
+    verb: 'patch',
+    resource: TemplateModel.plural,
+    namespace: template?.metadata?.namespace,
+  });
+
+  if (!template || !canUpdateLoaded || !canPatchLoaded) return false;
```

Comment:
now this should be if `(!template || canUpdateLoading || canPatchLoading) return false;`